src/views/games/Nertz.vue

```diff
@@ -1,10 +1,11 @@
 <template>
 	<v-container>
 		<v-row>
-			<v-col cols="4"></v-col>
-			<v-col cols="4"><p class="display-2 text-center">NERTZ</p></v-col>
 			<v-col cols="4"><Rules :game="this.gameTitle" /></v-col>
```

Comment:
Was this change made on purpose?

Reply:
Yes in my view it was on top of "Yu-Gi-Oh!"